src/sheldon.erl

```diff
@@ -0,0 +1,76 @@
+%%% @doc Main module for sheldon. Use this one from your own applications.
+%%%
+%%% Copyright 2016 Inaka &lt;hello@inaka.net&gt;
+%%%
+%%% Licensed under the Apache License, Version 2.0 (the "License");
+%%% you may not use this file except in compliance with the License.
+%%% You may obtain a copy of the License at
+%%%
+%%%     http://www.apache.org/licenses/LICENSE-2.0
+%%%
+%%% Unless required by applicable law or agreed to in writing, software
+%%% distributed under the License is distributed on an "AS IS" BASIS,
+%%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%%% See the License for the specific language governing permissions and
+%%% limitations under the License.
+%%% @end
+%%% @copyright Inaka <hello@inaka.net>
+%%%
+-module(sheldon).
+-author("Felipe Ripoll <ferigis@gmail.com>").
+
+%% API
+-export([
+          check/1
```

Comment:
Please, don't add empty lines before the first line when using `comma first` style.

```
-export([ check/1
        , check/2
        ]).
```